src/scss/_mixins.scss

```diff
@@ -80,16 +80,25 @@
 		display: block;
 	}
 
-	// Standardise outline styles
-	:focus {
+	// Standardise focus styles.
+	&:focus {
```

Comment:
I think we need to remove the ampersand as this is a base-level rule and base-level rules can't use a parent referencing selector because there is no parent selector.